src/it/scala/io/iohk/ethereum/transactionTest/util/DumpChainApp.scala

```diff
@@ -0,0 +1,98 @@
+package io.iohk.ethereum.transactionTest.util
+
+import akka.actor.ActorSystem
+import akka.agent.Agent
+import akka.util.ByteString
+import com.typesafe.config.ConfigFactory
+import io.iohk.ethereum.db.components.{SharedLevelDBDataSources, Storages}
+import io.iohk.ethereum.domain.{Blockchain, _}
+import io.iohk.ethereum.network.PeerManagerActor.PeerConfiguration
+import io.iohk.ethereum.network.p2p.messages.{PV62, PV63}
+import io.iohk.ethereum.network.{PeerManagerActor, loadAsymmetricCipherKeyPair}
+import io.iohk.ethereum.utils.{Config, NodeStatus, ServerStatus}
+import org.spongycastle.util.encoders.Hex
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.FiniteDuration
+
+object DumpChainApp {
+  // scalastyle:off
```

Comment:
Does this have any effect? Are we checking `it` code with scalastyle?
If it does, I'd prefer too see specific rules turned off or tweaked in the xml, rather than `// scalastyle:off`